classes/admin/class-page-settings.php

```diff
@@ -72,18 +73,19 @@ public function get_tabs_settings() {
 				'settings' => [
 					"has-page-{$page_type['slug']}" => [
 						'id'          => "has-page-{$page_type['slug']}",
-						'label'       => sprintf(
+						'title'       => $page_type['title'],
+						'description' => sprintf(
 							/* translators: The page name. */
 							esc_html__( 'Do your site have the "%s" page?', 'progress-planner' ),
 							esc_html( $page_type['title'] )
 						),
-						'description' => esc_html__( 'If you don\'t have this page yet, we can help you create it.', 'progress-planner' ),
 						'type'        => 'radio',
 						'options'     => [
-							'no'  => esc_html__( 'No', 'progress-planner' ),
-							'yes' => esc_html__( 'Yes', 'progress-planner' ),
+							'yes'            => esc_html__( 'Yes', 'progress-planner' ),
+							'no'             => esc_html__( 'No', 'progress-planner' ),
+							'not-applicable' => esc_html__( 'I don\'t need this page', 'progress-planner' ),
 						],
-						'value'       => empty( $type_pages ) ? 'no' : 'yes',
+						'value'       => $has_page_value,
```

Comment:
I think we can probably make this smarter... 🤔 

We can determine if the user has assigned a page to a specific page-type, so the yes/no can already be extrapolated by that. The only thing we should have to save is whether they need a page or not.

What do you think @ilicfilip?